src/core/utils/exchange-rates.ts

```diff
@@ -0,0 +1,7 @@
+import firebase from 'react-native-firebase';
+
+export const fetchExchangeRates = (callback: any) => {
```

Comment:
we need to check the performance with very large set of data. like exchange rates for 2k tokens.